src/views/OnTouchGui.jsx

```diff
@@ -31,52 +27,60 @@ import {
 
 const KEYCODE_D = 68;
 
-class OnTouchGui extends Component {
-  constructor(props) {
-    super(props);
-    this.checkedVersion = false;
-    this.state = {
-      developerMode: false,
-      currentStory: DefaultStory,
-      sliderStartStory: DefaultStory,
-    };
-
-    this.addStoryTree = this.addStoryTree.bind(this);
-    this.changeStory = this.changeStory.bind(this);
-    this.setStory = this.setStory.bind(this);
-    this.resetStory = this.resetStory.bind(this);
-    this.checkStorySettings = this.checkStorySettings.bind(this);
-    this.handleKeyPress = this.handleKeyPress.bind(this);
-    this.toggleDeveloperMode = this.toggleDeveloperMode.bind(this);
-  }
+function OnTouchGui({ luaApi, FetchData, StartConnection, anchorNode, changePropertyValue, triggerActionDispatcher,
```

Comment:
line width 

(see comment above)

Reply:
next line too

Reply:
We could also apply line width as a rule in the linter?